examples/02_control_flow/02_Loops.md

```diff
@@ -46,13 +54,13 @@ fun main(args: Array<String>) {
 
 </div>
 
-While and do-while constructs work similarly to most languages.
-
-1. Performs the block while the condition is true.
-2. Performs the block first, and then loops while evaluating the while condition.
+1. Executing the block while the condition is true.
+2. Executing the block first and then checking the condition.
 
 ### Iterators
 
+You can define own iterators in your classes by implementing the `iterator` operator in them.
```

Comment:
You can define your own iterators in your classes by implementing the `iterator` operator in them.